packages/flutter_tools/test/general.shard/utils_test.dart

```diff
@@ -69,6 +72,21 @@ baz=qux
       expect(snakeCase('ABC'), equals('a_b_c'));
     });
   });
+  test('Version.json info is correct',(){
```

Comment:
Create the flutter manifest once, then pass it into the flutter project. You also should use a `MemoryFileSystem.test` instead of the globals.fs, like:


```
final MemoryFileSystem fileSystem = MemoryFileSystem.test();
final FlutterManifest manifest = FlutterManfiest.createFromString('''
... content goes here
''');
final FlutterProject flutterProject = FlutterProject(fileSystem.current, manifest, logger: BufferLogger.test());


```